src/ScriptCs.Hosting/IVisualStudioSolution.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+
+namespace ScriptCs.Contracts
+{
+    public interface IVisualStudioSolution
+    {
+        void AddHeader();
+        void AddScriptcsProject(string scriptcsPath, string workingPath, string args, bool attach, string projectGuid);
+        void AddProject(string path, string name, string guid, string[] files);
+        void AddGlobalHeader(string projectGuid);
+        void AddGlobalNestedProjects(IList<Tuple<string, string>> nestedItems);
```

Comment:
can we get a better signature on a public member here? i.e. a descriptive type instead of a `list` of `Tuple` which is very difficult to comprehend